### DISCUSSION TITLE
Error in Xcode and white blank page when building it for ice or when launch on emulator

### DISCUSSION THREAD

**Discussion by ervinlihat** [quasarframework/quasar#9742] [CLI - Capacitor / Cordova modes] [answered]
Hi there, I am building a Quasar app for iOS and Android, using an existing API on another server. When running with "`quasar dev`" it works fine in the browser on localhost:8080. However when I move it to Xcode with "`quasar dev -m iOS --ide`" and try to run it from Xcode on any iPhone emulators it results in a white blank page, and I get the following two errors on the debug screen:

Failed to load webpage with error: The certificate for this server is invalid. You might be connecting to a server that is pretending to be “192.168.1.25” which could put your confidential information at risk.

and

[Process] 0x14d02d218 - [pageProxyID=11, webPageID=12, PID=81821] WebPageProxy::didFailProvisionalLoadForFrame: frameID = 3, domain = NSURLErrorDomain, code = -1202

What am I doing wrong? I feel like I am going something wrong, because like this the quasar still runs a server, so if I stop it from the Quasar CLI, then the emulated app won't connect to the server. What would be the proper way to build the iOS version in order to deploy it to Apple Market?

Thank you,
Best regards!

**Comment by ontwikkelfabriek** [comment, marked as answer]
@resolutionstudio 

> Failed to load webpage with error: The certificate for this server is invalid. You might be connecting to a server that is pretending to be “192.168.1.25” which could put your confidential information at risk.

This is probably about the Quasar/Webpack connection (for live reload ect ) that the `dev` command will spawn between Quasar's **dev** server and the app.

What I do is this: instead of "quasar dev -m iOS --ide" build the project for IOS , then go to the **generated** xcode project and open the `workspace` **instead** of the `project` in xcode. Then run/start the project from inside xcode( you can debug with xcode or safari).

**Reply by ervinlihat** [reply]
@ontwikkelfabriek thank you for help, it solved my problem. Big thanks!
Ervin
